README.md

```diff
@@ -130,7 +130,7 @@ The rest
 
 ## Integrations
 
-When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and Twitter to post new-user announcements. 
+When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and use the X social-meida platform to post new-user announcements (now requires payment). 
```

Comment:
```suggestion
When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and Twitter to post new-user announcements.
```
We don't need to say that it requires payment here. It feels awkward to place that in an intro to the integrations.